src/express/common/config-utils.js

```diff
@@ -107,6 +107,10 @@ export function splitToArray(value) {
     return value.split(' ').join('').split(",")
 }
 
+export function splitIp(value) {
```

Comment:
As we use this only with the `hostIp`, can we rename the method to `splitArrayAndGetHostIp` and make it return `value.split("@")[0]`?
Also, please introduce an error handling check.
If value passed in is not an array or is undefined, the program should log an error and exit with status code `1`

Reply:
Done: [9357450](https://github.com/maticnetwork/matic-cli/pull/86/commits/9357450e975cf3541ff7bdf3e9d0eb979b88726f)